pages/common/kubectl-run.md

```diff
@@ -0,0 +1,24 @@
+# kubectl run
+
+> Run pods in Kubernetes. Specifies pod generator to avoid deprecation error in some K8S versions.
+> More information: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#run>.
+
+- Start an nginx pod and expost port 80
+
+`kubectl run --generator=run-pod/v1 nginx --image=nginx --port 80`
+
+- Run an nginx pod, setting the TEST_VAR environment variable:
+
+`kubectl run --generator=run-pod/v1 nginx --image=nginx --env="TEST_VAR=testing"`
+
+- Show API calls that would be made to create an nginx container:
+
+`kubectl run --generator=run-pod/v1 nginx --image=nginx --dry-run`
+
+- Run a one-off Ubuntu pod interactively, never restart it, and remove it when it exits:
```

Comment:
I am not sure about the comma here, I might not be correct here .. i am going with my gut.
Please correct me if ....
```suggestion
- Spawn a Ubuntu pod interactively, never restart it and remove it when it exits:
```

Reply:
I think this example is also fine as-is.

Reply:
If you remove `one-off` you have to change `a Ubuntu` to `an Ubuntu`.

Reply:
Also, if you remove `one-off` you have to change `a Ubuntu` to `an Ubuntu`.

Reply:
> I think this example is also fine as-is. Any particular reason for `Spawn` over `Run`?

In my understanding a pod is `spawned`. What i am looking for here is like .. we spawn a process and run a command.
Although a nitpick, but my gut disagrees to run a `pod`. I would more like say.. 
Run a Command; Spawn a process .. but still i am open :)

Reply:
The linked documentation page (https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#run) uses the terminology `run` for pods, so I think it would make sense to remain consistent with that in the documentation here.

Reply:
Ah, I see @cristamd. In that case, we should definitely use `Run` instead of `Spawn`.